src/sentry/event_manager.py

```diff
@@ -2377,14 +2383,15 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
 
 
 @metrics.wraps("performance.performance_issue.should_create_group", sample_rate=1.0)
-def should_create_group(client: Any, grouphash: str) -> bool:
+def should_create_group(client: Any, grouphash: str, type: GroupType) -> bool:
     times_seen = client.incr(f"grouphash:{grouphash}")
     metrics.incr(
         "performance.performance_issue.grouphash_counted",
         tags={"times_seen": times_seen},
```

Comment:
add a tag here for `GroupType` (human readable string of the enum, if possible)

Reply:
good idea for sure 👍

Reply:
added 👍